src/Microsoft.AspNet.Mvc.ModelBinding/BinderMetadata/FromServicesAttribute.cs

```diff
@@ -8,8 +8,60 @@ namespace Microsoft.AspNet.Mvc
 {
     /// <summary>
     /// This attribute is used on action parameters or model properties to indicate that
-    /// they will be bound using service provider.
+    /// they will be bound with the <see cref="ServicesModelBinder"/> using the <see cref="IServiceProvider"/>.
     /// </summary>
+    /// <example>  
```

Comment:
The example has too many unrelated items, which makes it very long and not very useful in an intellisense window.
It is also hard to keep this sample in sync with the real recommended user code if for example registration APIs change over time.

We are going to have a short discussion in the team, and get back to you with recommendations.